drivers/input/rmi4/rmi_2d_sensor.h

```diff
@@ -66,6 +66,8 @@ struct rmi_2d_sensor {
 	u8 y_mm;
 	enum rmi_reg_state dribble;
 	enum rmi_reg_state palm_detect;
+	bool bSupportQueryDPM;
```

Comment:
The name here don't follow the kernel's naming convention, maybe change to support_dpm_query and dpm_resolution.